integration-ddb-to-lambda/example.php

```diff
@@ -0,0 +1,48 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+<?php
+
+# using bref/bref and bref/logger for simplicity
+
+use Bref\Context\Context;
+use Bref\Event\DynamoDb\DynamoDbEvent;
+use Bref\Event\DynamoDb\DynamoDbHandler;
+use Bref\Logger\StderrLogger;
+
+require __DIR__ . '/vendor/autoload.php';
+
+class Handler extends DynamoDbHandler
+{
+    private StderrLogger $logger;
+
+    public function __construct(StderrLogger $logger)
+    {
+        $this->logger = $logger;
+    }
+
+    /**
+     * @throws JsonException
+     * @throws \Bref\Event\InvalidLambdaEvent
+     */
+    public function handleDynamoDb(DynamoDbEvent $event, Context $context): void
+    {
+        $this->logger->info("Processing DynamoDb table items");
+        $records = $event->getRecords();
+
+        foreach ($records as $record) {
+            $keys = $record->getKeys();
+            $old = $record->getOldImage();
+            $new = $record->getNewImage();
```

Comment:
Can you have this section match the other examples in this directory? i.e. in Python

```
def log_dynamodb_record(record):
    print(record['eventID'])
    print(record['eventName'])
    print(f"DynamoDB Record: {json.dumps(record['dynamodb'])}")
```

(Instead of print you can use `$this->logger->info`).